backend/src/main/java/ca/gov/dtsstn/cdcp/api/controller/SubscriptionControllerV1.java

```diff
@@ -15,7 +16,7 @@
 public class SubscriptionControllerV1 {
 
 	@GetMapping({ "/users/{userId}/subscriptions" })
-	public List<Subscription> getSubscriptions(@PathVariable String userId) {
+	public Collection<Subscription> getSubscriptions(@PathVariable String userId) {
```

Comment:
Since it returns a specific implementation of the Collection interface (List), it's more appropriate to declare the return type as List<Subscription> to reflect the specific type being returned accurately.

```suggestion
	public List<Subscription> getSubscriptions(@PathVariable String userId) {
```